### PR TITLE
Merging master jdk-15.0.2+7 into openj9-staging 

### PR DESCRIPTION
**Pull request by pshipton**
Fixed merge conflict in make/Main.gmk

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins compile zlinux jdk15

File: make/Main.gmk [hunk 1]
```diff
@@ -1122,7 +1136,9 @@ endif
 docs-image: docs-jdk
 
 # This target builds the test image
-test-image: prepare-test-image test-image-jdk-jtreg-native test-image-demos-jdk
+test-image: prepare-test-image test-image-jdk-jtreg-native \
+    test-image-demos-jdk test-image-libtest-jtreg-native \
+    test-image-lib-native
```
Comment (JasonFengJ9) [inline make/Main.gmk:1141]: Are we getting rid of OpenJ9 patches?

Comment (keithc-ca) [reply make/Main.gmk:1141]: This looks to have come straight from openjdk.

Comment (JasonFengJ9) [reply make/Main.gmk:1141]: https://github.com/ibmruntimes/openj9-openjdk-jdk15/commit/1a0d15411f5042be9fb78ede85c4a272d175a544#diff-db3facd0a5ba03518baca7e9a30465068f70b3ea29dd3988f6dbaf3b246280d1
Note the target `test-image-libtest-jtreg-native`.

Comment (pshipton) [reply make/Main.gmk:1141]: Not intentionally.  I don't see any OpenJ9 patch to these lines. The change here is reformatting, plus adding `test-image-lib-native` which comes from the OpenJDK 15.0.2 changes. The next few lines after this, which were added for OpenJ9, ~will discard `test-image-lib-native`, but I'm not sure it's needed for OpenJ9 anyway~ includes `test-image-libtest-jtreg-native` for non-OpenJ9. The PR build did fail, ~but it looks unrelated~.

Comment (pshipton) [reply make/Main.gmk:1141]: I see it now, updating ...

Comment (pshipton) [reply make/Main.gmk:1141]: Updated

**Review by JasonFengJ9 — APPROVED** [review]
There was a build error
```
16:51:21  gensrc/GensrcJvmti.gmk:35: *** SRC specified to SetupJavaCompilation BUILD_JVMTI_TOOLS contains missing directory >/home/jenkins/workspace/Build_JDK15_s390x_linux_Personal/src/hotspot/share/prims<.  Stop.
16:51:21  gmake[2]: *** [hotspot-server-gensrc] Error 1
```
It is not related to the merging conflict, I think it can be handled in abuild if it occurs again.

**Comment by pshipton** [comment]
jenkins compile zlinux jdk15

**Comment by pshipton** [comment]
Seems the PR failure was caused by the merge problem. I've started another build to confirm it's working now.

**Review by keithc-ca — APPROVED** [review]

**Comment by pshipton** [comment]
The PR build passed this time.

**Pull request merged**